.travis.yml

```diff
@@ -6,20 +6,51 @@ matrix:
       dist: trusty
       sudo: required
       compiler: gcc
-      env: COVERAGE=1 CMAKE_OPTIONS="-DCMAKE_BUILD_TYPE=Release -DEXIV2_ENABLE_VIDEO=ON -DEXIV2_ENABLE_WEBREADY=ON -DEXIV2_BUILD_UNIT_TESTS=ON -DBUILD_WITH_COVERAGE=ON" # All enabled + Coverage
+      env: COVERAGE=1 CMAKE_OPTIONS="-DCMAKE_BUILD_TYPE=Release -DEXIV2_ENABLE_VIDEO=ON -DEXIV2_ENABLE_WEBREADY=ON -DEXIV2_BUILD_UNIT_TESTS=ON -DBUILD_WITH_COVERAGE=ON -DEXIV2_TEAM_USE_SANITIZERS=ON" # All enabled + Coverage
+
+    - os: linux
+      dist: trusty
+      sudo: required
+      compiler: gcc-8
+      addons:
+        apt:
+          update: true
+          sources:
+          - sourceline: 'ppa:ubuntu-toolchain-r/test'
+          packages:
+            - g++-8
```

Comment:
Ey @D4N, I think it would be better to isolate this change about CI in another different PR. Or do you have a strong reason to do it also in this PR?

Reply:
Yes, gcc 4.8 does not ship with UBSAN and clang's UBSAN detects different kinds of undefined behavior (the `getEnv` stuff was only detected by clang).